data/router.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const UserDB = require('./helpers/userDb.js');
+const routesUsers = require('./routesUser');
+const routesPosts = require('./routesPosts');
+
+const router = express.Router();
+
```

Comment:
if you were going to use helmet for security purposes, it would be inserted here as the first middleware

Reply:
Ohhh, did I not do that right? I loaded helmet at server.js. Sending you a message on slack about this.